library/wizard/src/modules/Wizard.rb

```diff
@@ -606,6 +606,22 @@ def OpenLeftTitleNextBackDialog
       nil
     end
 
+    def OpenWithLayout(layout)
+      UI.SetProductLogo(layout.banner?)
+
+      if layout.mode.steps?
+        public_send(:OpenNextBackStepsDialog)
```

Comment:
as you call it here in Wizard, public_send is not needed and you can simply use `OpenNextBackStepsDialog`

Reply:
It complains because missing constant when calling `OpenNextBackStepsDialog` directly. Using `self.OpenNextBackStepsDialog` instead.